zib/StructureDefinitions/zib-AnatomicalLocation.xml

```diff
@@ -0,0 +1,75 @@
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An anatomical location specifies the location (e.g. foot) and laterality (e.g. left) of a body part." />
+  <purpose value="This CodeableConcept resource represents Location concept of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) AnatomicalLocation v1.0.3 (2024)](https://zibs.nl/wiki/AnatomicalLocation-v1.0.3(2024EN)). It provides a BodyStructure reference that caputures the Laterality concept.&#xD;&#xA;&#xD;&#xA;Profilers can use this profile in places where zib AnatomicalLocation is instantiated." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-anatomicallocation-v1.0.3-2024EN" />
+    <uri value="https://zibs.nl/wiki/AnatomicalLocation-v1.0.3(2024EN)" />
+    <name value="zib AnatomicalLocation-v1.0.3(2024EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="true" />
+  <type value="CodeableConcept" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CodeableConcept" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CodeableConcept">
+      <path value="CodeableConcept" />
+      <short value="Location" />
+      <definition value="Localisation on/in the body." />
+      <comment value="The zib AnatomicalLocation is mapped to both a profile on BodyStructure (&lt;http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation-BodyStructure&gt;) and this CodeableConcept profile. The CodeableConcept handles the representation of the location, while the BodyStructure profile covers both location and laterality.&#xD;&#xA;&#xD;&#xA;In cases where only the location is required, or where laterality is embedded within the location code, this CodeableConcept can be used within the resource. However, when laterality needs to be explicitly recorded, the BodyStructure resource should be used, referenced through the BodyStructure Reference within this CodeableConcept. If the BodyStructure reference is used, this CodeableConcept should either remain empty or align with the BodyStructure. element.&#xD;&#xA;&#xD;&#xA;**Note 1:** Receiving systems must support both this CodeableConcept and the BodyStructure resource, as either mechanism may be employed. The rationale for this is that, in many cases, only the location will be required, simplifying the FHIR recording process.&#xD;&#xA;&#xD;&#xA;**Note 2:** Since this is a datatype profile, the binding has been applied to the `.coding` element, as it is not possible to bind the ValueSet to the root element." />
```

Comment:
This comment should be placed in `StructureDefinition.purpose`, but terser comment should be placed here about when to use the one and when to use the other.

Reply:
Moved some of this to the SD purpose and made this comment more concise.